test/Atmos/Parameterizations/TurbulenceConvection/BOMEX.jl

```diff
@@ -0,0 +1,47 @@
+using Test
+
+using CLIMA.TurbulenceConvection
+
+const test_data_dir = joinpath(pwd(),"output","TestData")
+mkpath(test_data_dir)
+
+#### Accepting a new solution
+# After a careful review of ALL of the solution results,
+# set `accept_new_solution = true`, and run once. Then,
+# reset to `false` before committing. This parameter should
+# never be `true` for commits.
+const accept_new_solution = false
+
+tc = TurbulenceConvection
+@testset "Integration test: EDMF equations (BOMEX)" begin
+  grid, q, tmp = tc.EDMF.run(tc.EDMF.BOMEX())
+
+  if accept_new_solution || !isfile(joinpath(test_data_dir, "q_expected.csv"))
+    export_state(q, grid, test_data_dir, "q_expected.csv")
+  end
+  if accept_new_solution || !isfile(joinpath(test_data_dir, "tmp_expected.csv"))
+    export_state(tmp, grid, test_data_dir, "tmp_expected.csv")
+  end
```

Comment:
This was a hack for refactoring. We should change this to use `DataDeps.jl` or something, as this will _always_ pass on Azure (whether the results are correct or not).